### PR TITLE
Add "The Super Programmer"

### PR DESCRIPTION
**Pull request by keyvank**

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -790,6 +790,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [The Black Art of Programming](http://self.gutenberg.org/wplbn0002828847-the-black-art-of-programming-by-mcilroy-mark.aspx?) - Mark McIlroy
 * [The Craft of Programming](https://kilthub.cmu.edu/articles/The_Craft_of_Programming/6610514) - John C. Reynolds
 * [The Nature of Code](https://natureofcode.com/book) - Daniel Shiffman (HTML)
+* [The Super Programmer / Building software that run our world!](https://github.com/keyvank/tsp) - - Keyvan Kambakhsh (PDF, HTML)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:793]:
```suggestion
* [The Super Programmer](https://github.com/keyvank/tsp) - Keyvan Kambakhsh (PDF, HTML)
```
As you have different subtitles in different places, just use the title.

**Comment by eshellman** [comment]
also, you may want to add the *in progress* tag - read docs/CONTRIBUTING.md guidelines.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by keyvank** [comment]
Thanks!

**Pull request merged**